### DISCUSSION TITLE
Adding and updating opening stocks without purchases

### DISCUSSION THREAD

**Discussion by boudo2017** [Blair2004/NexoPOS#516] [General]
Hi,
It is necessary to add this feature:"
Adding and updating opening stocks without purchases."
regards

**Comment by Blair2004** [comment]
![screenshot-dev-v4 nexopos com-2022 01 17-09_48_28](https://user-images.githubusercontent.com/5265663/149737275-394282b5-e4bb-4b27-9864-bc60615bf323.jpg)
The stock can be adjusted here.
